src/lib/intent/composed-intent.ts

```diff
@@ -36,6 +37,15 @@ export default class ComposedIntent implements Intent {
   }
 
   // TODO: adjust interface so we don't have to implement all this
+  getSaveAction (): APIAction {
+    return null
+  }
+  getPublishAction (): APIAction {
```

Comment:
This just mirrors the interface from above and therefore isn't used as well.

Reply:
Ah yes, remains from an attempt to make the save content type and publish content type actions generic.
I'll get rid of these